src/main/java/Task.java

```diff
@@ -0,0 +1,46 @@
+public class Task {
+    private String desc;
+    private int status; // 0 incomplete, 1 complete
+
+    public Task(String d, int s) {
+        this.desc = d;
+        this.status = s;
+    }
+
+    public void update(int i) {
```

Comment:
maybe the name for this method can be setStatus? Just a suggestion.

Reply:
Good suggestion, will implement. Thank you!